install-clients.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
I would add

set -ex

to the script to halt on errors. (https://serverfault.com/a/391259)